runtime/pkg/email/templates/scheduled_report.mjml

```diff
@@ -26,7 +32,7 @@
 
         <mj-spacer height="20px" />
 
-        <mj-button background-color="#ECF0FF" color="#3524C7" font-weight="bold" href="{{ .OpenLink }}">
+        <mj-button css-class="{{ if not .OpenLink }}hidden{{ end }}" background-color="#ECF0FF" color="#3524C7" font-weight="bold" href="{{ .OpenLink }}">
```

Comment:
Just double checking – did you send test emails both with and without the open link? Just to make sure it still shows for normal users.